src/Microsoft.PowerApps.TestEngine/Reporting/TestReporter.cs

```diff
@@ -104,6 +110,15 @@ public void EndTestRun(string testRunId)
 
             testRun.Times.Finish = DateTime.Now;
             testRun.ResultSummary.Outcome = "Completed";
+
+            // Update the ResultSummary Output
+            _updateResultSummaryOutPut(testRun);
+        }
+
+        private void _updateResultSummaryOutPut(TestRun testRun)
+        {
+            var resultOutputMessage = $"{{ \"AppURL\": {testRunAppURL}, \"TestResults\": {testResultsDirectory}}}";
```

Comment:
`{testRunAppURL}` inside `resultOutputMessage` should be a quoted string, like `"{testRunAppURL}"` without quotes for the values it will be an invalid json.
this one also has the double enclosing curly braces.